app/models/concerns/syncable.rb

```diff
@@ -4,33 +4,19 @@ module Syncable
   extend ActiveSupport::Concern
 
   included do
-    scope :maxed_out, -> { where('attempts > ?', NB.attempts.to_i).order('updated_at') }
-    scope :need_syncdown, -> { where('dirty = ? AND attempts <= ? AND try_again_at < ?', true, NB.attempts.to_i, Time.now).order('updated_at') }
+    scope :need_syncdown, -> { where('dirty = ?', true).order('updated_at') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.